packages/flutter/test/material/slider_test.dart

```diff
@@ -975,6 +1005,82 @@ void main() {
     }
   });
 
+  testWidgets('Slider value indicator respects bold text', (WidgetTester tester) async {
+      final Key sliderKey = UniqueKey();
+      double value = 0.0;
+      final List<InlineSpan> log = <InlineSpan>[];
+      final LoggingValueIndicatorShape loggingValueIndicatorShape = LoggingValueIndicatorShape(log);
+
+      Widget buildSlider({
+        bool boldText = false,
+      }) {
+        return MaterialApp(
+          theme: ThemeData(useMaterial3: false),
```

Comment:
Unless the fix is specific to M2, we could be testing for M3 by default and remove this line.